tools/ops/mugc.py

```diff
@@ -145,6 +155,9 @@ def setup_parser():
     parser.add_argument(
         '-c', '--config', dest="config_files", nargs="*", action='append',
         help="Policy configuration files(s)", default=[])
+    parser.add_argument(
+        "--targeted", action="store_true", default=False,
```

Comment:
not in love with the name, but drawing blanks on the moment on something better. the default is effectively --absent so this is perhaps --present ?

Reply:
Agreed.  I was at a loss for naming as well